core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -2232,7 +2232,7 @@ public void doCommitDDL(final TransactionState transactionState) throws Unsucces
                         if (retry) 
                             retrySleep = retry(retrySleep);
                     }
-                } while (retry == true);
+                } while (retry && retryCount++ <= RETRY_ATTEMPTS);
```

Comment:
Thanks @DaveBirdsall. I will consider making this change when I need to change this file again